modules/subdyn/src/SD_FEM.f90

```diff
@@ -436,11 +447,13 @@ SUBROUTINE SD_Discrt(Init,p, ErrStat, ErrMsg)
    INTEGER                       :: NNE      ! number of nodes per element
    INTEGER                       :: MaxNProp
    REAL(ReKi), ALLOCATABLE       :: TempProps(:, :)
+   REAL(ReKi), ALLOCATABLE       :: TempPropsX(:, :)
    INTEGER, ALLOCATABLE          :: TempMembers(:, :)
    INTEGER                       :: knode, kelem, kprop, nprop
+   INTEGER                       :: COSM_ind
```

Comment:
I would maybe use `iDirCos` and add a comment  `!< index of directional cosine matrix (-1 if not used)`